<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>19</y>
      <width>771</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="pageInput">
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>68</x>
        <y>70</y>
        <width>611</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="tanggalLabel">
         <property name="text">
          <string>Tanggal</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDateEdit" name="dateEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="foodLabel">
         <property name="text">
          <string>Makanan/Minuman</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_food"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="kategoriLabel">
         <property name="text">
          <string>Kategori</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox_category">
         <item>
          <property name="text">
           <string>Breakfast</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Lunch</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Snack</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dinner</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Drink</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="kaloriLabel">
         <property name="text">
          <string>Kalori (kkal)</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="spinBox_calories"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="catatanLabel">
         <property name="text">
          <string>Catatan</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_note"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>370</y>
        <width>441</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btnAdd">
           <property name="text">
            <string>Simpan</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnUpdate">
           <property name="text">
            <string>Ubah</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDelete">
           <property name="text">
            <string>Hapus</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClear">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="labelTambah">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>551</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Tambah Data Diet</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageHistory">
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>751</width>
        <height>401</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnExport">
         <property name="text">
          <string>Ekspor CSV</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="labelRiwayat">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>0</y>
        <width>461</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Riwayat Konsumsi</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="filterSearchWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>751</width>
        <height>40</height>
       </rect>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>751</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="lineEdit_search">
          <property name="placeholderText">
           <string>Cari makanan / catatan...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_filter">
          <item>
           <property name="text">
            <string>Semua</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Breakfast</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Lunch</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Snack</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dinner</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Drink</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="pageStats">
     <widget class="QLabel" name="labelStats">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>0</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Statistik Kalori</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>541</width>
        <height>307</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCalendarWidget" name="calendarWidget"/>
       </item>
       <item>
        <widget class="QLabel" name="labelTotalKalori">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelTotalEntri">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelChart">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSimpan"/>
    <addaction name="actionExport"/>
    <addaction name="actionKeluar"/>
   </widget>
   <widget class="QMenu" name="menuNavigasi">
    <property name="title">
     <string>Navigasi</string>
    </property>
    <addaction name="actionInput"/>
    <addaction name="actionRiwayat"/>
    <addaction name="actionStatistik"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionTentang"/>
    <addaction name="actionBantuan"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuNavigasi"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionSimpan">
   <property name="text">
    <string>Simpan</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Ekspor CSV</string>
   </property>
  </action>
  <action name="actionKeluar">
   <property name="text">
    <string>Keluar</string>
   </property>
  </action>
  <action name="actionInput">
   <property name="text">
    <string>Input Data</string>
   </property>
  </action>
  <action name="actionRiwayat">
   <property name="text">
    <string>Riwayat</string>
   </property>
  </action>
  <action name="actionStatistik">
   <property name="text">
    <string>Statistik</string>
   </property>
  </action>
  <action name="actionTentang">
   <property name="text">
    <string>Tentang Aplikasi</string>
   </property>
  </action>
  <action name="actionBantuan">
   <property name="text">
    <string>Bantuan</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
